homeassistant/components/media_player/squeezebox.py

```diff
@@ -293,64 +303,64 @@ def supported_media_commands(self):
 
     def turn_off(self):
```

Comment:
If we're updating for #4995 we should convert these where appropriate.

Which reminds me that we need to get that merged!